server/jobs/project_command_output_handler.go

```diff
@@ -177,8 +180,10 @@ func (p *AsyncProjectCommandOutputHandler) GetReceiverBufferForPull(jobID string
 }
 
 func (p *AsyncProjectCommandOutputHandler) GetJob(jobID string) Job {
-	job, _ := p.jobStore.Get(jobID)
-	return job
+	if job, _ := p.jobStore.Get(jobID); job != nil {
+		return *job
```

Comment:
Why is job store returning a pointer if you're going to dereference it over here?

Reply:
Where is `GetJob` is being used? I searched in the codebase and seems it is only used by tests.

Reply:
It is used when we register new connections in the projectCommandOutputHandler() 
https://github.com/lyft/atlantis/blob/fa0e909c8ef8928d8dc68c369c8096be50ace354/server/jobs/project_command_output_handler.go#L131

Reply:
That is `JobStore.Get(jobID)`, not `AsyncProjectCommandOutputHandler.GetJob`.

Reply:
Oh I got confused! yeah it's only being used for testing purposes. I have removed that method and updated the tests!